src/registry.cpp

```diff
@@ -116,9 +116,28 @@ bool Registry::logout() {
     return UserProfile::get()->getAuthManager()->deleteCredentials(url);
 }
 
+void smartDisplay(std::ostream& stream, double number) {
+
+    stream << std::setprecision(4);
+
+    if (number < 1024) {
+        //
+        stream << number << " bytes";
+    } else if (number < 1048576)
```

Comment:
Use `io::bytesToHuman(number)` :+1:

Reply:
Done in https://github.com/DroneDB/DroneDB/pull/190/commits/a3950f1dd6e847d0cb56eafdbe2d26ed32ad5903